lib/react.dart

```diff
@@ -523,12 +523,32 @@ abstract class Component2 implements Component {
   /// This method is invoked before any instances are created and thus cannot rely on [props]. In addition, be aware
   /// that any complex objects returned by `defaultProps` will be shared across instances, not copied.
   ///
-  /// See: <https://reactjs.org/docs/react-component.html#defaultprops>
```

Comment:
I see that you removed these links, but I think it's good to have links back to the official React docs where possible. Could you reinstate them (and potentially use the newer ones I posted in a comment for `initialState`?